galley/pkg/kube/source/listener.go

```diff
@@ -153,10 +153,12 @@ func (l *listener) handleEvent(c resource.EventKind, obj interface{}) {
 		var tombstone cache.DeletedFinalStateUnknown
 		if tombstone, ok = obj.(cache.DeletedFinalStateUnknown); !ok {
 			scope.Errorf("error decoding object, invalid type: %v", reflect.TypeOf(obj))
+			recordHandleEventError()
```

Comment:
Could we include the error as well here? The total number of errors is bounded by the number of supported types (low tens).